test/multi_search_test.rb

```diff
@@ -33,6 +33,15 @@ def test_misspellings_below_unmet
     assert_equal ["abc", "abd"], products.map(&:name)
   end
 
+  def test_misspellings_below_with_errored_query
+    store_names ["Product A"]
+    clause_limit = 1_024
+    search_string = (["Z"] * (clause_limit + 1)).join(" ")
+    products = Product.search(search_string, misspellings: {below: 1})
+    Searchkick.multi_search([products])
+    assert products.error
```

Comment:
The real goal of this test is not so much to `assert products.error` (though that is good to test), but rather to implicitly assert that `Searchkick.multi_search([products])` does not raise an exception, which it currently will on `master` if this test is executed without the change to `lib/searchkick/query.rb` that is included in this PR.